extensions/VisualEditor/lib/ve/src/ui/actions/ve.ui.WindowAction.js

```diff
@@ -59,7 +59,7 @@ ve.ui.WindowAction.prototype.open = function ( name, data ) {
 
 	surface.getView().deactivate();
 
-	if ( windowType === 'toolbar' || name === 'wikiaSourceMode' || name === 'wikiaTemplateInsert' || name === 'transclusion' ) {
+	if ( windowType === 'toolbar' || name === 'wikiaSourceMode' || name === 'wikiaInfoboxInsert' || name === 'wikiaTemplateInsert' || name === 'transclusion' ) {
```

Comment:
if these values are consts maybe we should move them to an `array` and use something like `Array.prototype.some()` to test this condition?